wandb/cli/cli.py

```diff
@@ -849,9 +856,18 @@ def _parse_settings(settings):
 
 def _check_launch_imports():
     req_string = 'wandb launch requires additional dependencies, install with pip install "wandb[launch]"'
-    _ = util.get_module("docker", required=req_string,)
-    _ = util.get_module("repo2docker", required=req_string,)
-    _ = util.get_module("chardet", required=req_string,)
+    _ = util.get_module(
+        "docker",
+        required=req_string,
+    )
```

Comment:
We shouldn't need the `docker` library as a dependency anymore.  We just shell out to use docker.